### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by nicktate**
Once https://github.com/containership/containership.cli/pull/27 is merged, I will rebase this PR. 

To see a current iteration live, you can view https://nicktate.github.io/containership.cli/ . 

@normanjoyner @ashleyschuett @mikestoltz

### CODE REVIEW

File: docs/_pages/quick-start-guide.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+permalink: '/docs/quick-start-guide/'
+---
+---
+
+# Quick Start Guide
+
+First install the command line client by following on the of the installation methods
```
Comment (phildougherty) [inline docs/_pages/quick-start-guide.md]: s/following on the/following one of the/

File: docs/_pages/quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+┌──────────────────────────────────────┬───────────────────────────┐
+│ ID                                   │ ORGANIZATION              │
+├──────────────────────────────────────┼───────────────────────────┤
+│ 00000000-0000-0000-0000-000000000000 │ sample-org                │
+└──────────────────────────────────────┴───────────────────────────┘
+~~~
+
+Then, copy the ID and run the command:
+
+`csctl cloud org use <org_id}`
```
Comment (phildougherty) [inline docs/_pages/quick-start-guide.md]: } should be >

File: docs/_pages/commands/info.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+~~~
+csctl info
+
+┌─────────────────────────────┬──────────────────────────────────────┐
+│ CPUS (USED / TOTAL)         │ 0.50 / 2.00                          │
+├─────────────────────────────┼──────────────────────────────────────┤
+│ MEMORY (USED / TOTAL)       │ 544 MB / 1901 MB                     │
+├─────────────────────────────┼──────────────────────────────────────┤
+│ APPLICATIONS                │ containership-logs                   │
```
Comment (normanjoyner) [inline docs/_pages/commands/info.md]: SERVICES

File: commands/container.js [hunk 4]
```diff
@@ -85,11 +86,11 @@ module.exports.commands.push({
                 .value();
 
             if(data.length === 0) {
-                return console.error('No containers exist on the cluster given your host and application filters.');
+                return logger.error('No containers exist on the cluster given your host and application filters.');
```
Comment (normanjoyner) [inline commands/container.js]: service

File: commands/container.js [hunk 5]
```diff
@@ -53,7 +54,7 @@ module.exports.commands.push({
                 });
 
                 if(!oneExists) {
-                    return console.error(`None of the '${app_names.toString()}' applications exist on the cluster, could not retrieve containers!`);
+                    return logger.error(`None of the '${app_names.toString()}' applications exist on the cluster, could not retrieve containers!`);
```
Comment (normanjoyner) [inline commands/container.js]: services

File: commands/container.js [hunk 6]
```diff
@@ -250,19 +251,19 @@ module.exports.commands.push({
...
+                    return logger.error('Could not fetch containers!');
                 }
 
                 if(!response.statusCode === 204) {
-                    return console.error(`Failed to remove container ${argv.container_id} from application ${app_name}!`);
+                    return logger.error(`Failed to remove container ${argv.container_id} from application ${app_name}!`);
                 }
 
-                return console.info(`Successfully removed container ${argv.container_id} from application ${app_name}!`);
+                return logger.info(`Successfully removed container ${argv.container_id} from application ${app_name}!`);
```
Comment (normanjoyner) [inline commands/container.js]: service

File: commands/container.js [hunk 7]
```diff
@@ -250,19 +251,19 @@ module.exports.commands.push({
...
 
             return request.delete(`applications/${app_name}/containers/${argv.container_id}`, {}, (err, response) => {
                 if(err) {
-                    return console.error('Could not fetch containers!');
+                    return logger.error('Could not fetch containers!');
                 }
 
                 if(!response.statusCode === 204) {
-                    return console.error(`Failed to remove container ${argv.container_id} from application ${app_name}!`);
+                    return logger.error(`Failed to remove container ${argv.container_id} from application ${app_name}!`);
```
Comment (normanjoyner) [inline commands/container.js]: service

File: docs/_pages/commands/client-plugin-sync.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+command line client configuration. Sync will iterate through all the plugins in your `.containership` plugin directory
+and attempt to load that into the client configuration.
+
+~~~
+$ ./bin/cli.js client-plugin sync
+
+┌────────────────────────────┬─────────┬──────────────────────────────────────────────────┬──────────────────────────────────────────────────┐
+│ NAME                       │ VERSION │ DESCRIPTION                                      │ PATH                                             │
+├────────────────────────────┼─────────┼──────────────────────────────────────────────────┼──────────────────────────────────────────────────┤
+│ containership.plugin.cloud │ 0.14.0  │ Containership cloud management client.           │ /Users/nick/.containership/plugins/containership │
```
Comment (normanjoyner) [inline docs/_pages/commands/client-plugin-sync.md]: Cloud

File: docs/_pages/commands/client-plugin-list.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+The {{ page.command-name }} allows you to view a list of all the configured cli plugins installed.
+</p>
+
+~~~
+csctl client-plugin list
+
+┌────────────────────────────┬─────────┬──────────────────────────────────────────────────┬──────────────────────────────────────────────────┐
+│ NAME                       │ VERSION │ DESCRIPTION                                      │ PATH                                             │
+├────────────────────────────┼─────────┼──────────────────────────────────────────────────┼──────────────────────────────────────────────────┤
+│ containership.plugin.cloud │ 0.14.0  │ Containership cloud management client.           │ /Users/nick/.containership/plugins/containership │
```
Comment (normanjoyner) [inline docs/_pages/commands/client-plugin-list.md]: Cloud

File: docs/_pages/commands/svc-delete.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+permalink: '/docs/commands/svc-delete'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to remove an service from your cluster.
+</p>
+
+> If this is a Containership cloud cluster, it must not be `locked`
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-delete.md]: Cloud

File: docs/_pages/plugins/cloud/cloud-account-edit.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+---
+command-name: 'cloud account edit'
+permalink: '/docs/plugins/cloud/cloud-account-edit'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to edit details about your containership cloud account.
```
Comment (normanjoyner) [inline docs/_pages/plugins/cloud/cloud-account-edit.md]: Containership Cloud

File: docs/_pages/quick-start-guide.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+├──────────────────────────────────────┼───────────────────────────┼─────────────┼────────────────┼────────────┼───────────┤
+│ 00000000-0000-0000-0000-000000000000 │ sample-cluster            │ dev         │ Digital Ocean  │ 2          │ 4         │
+└──────────────────────────────────────┴───────────────────────────┴─────────────┴────────────────┴────────────┴───────────┘
+~~~
+
+Similar to the `org use` command, you can set the active cluster with:
+
+`csctl cloud cluster use <cluster_id>`
+
+Once set, it will have automatically created a `remote` for you authenticated with your Containership cloud account.
```
Comment (normanjoyner) [inline docs/_pages/quick-start-guide.md]: Cloud

File: docs/_pages/plugins/cloud/cloud-account.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
+---
+command-name: 'cloud account'
+permalink: '/docs/plugins/cloud/cloud-account'
+---
+
+<h2> Overview </h2>
+
+<p>
+The {{ page.command-name }} command allows you to display and configure your containership cloud account.
```
Comment (normanjoyner) [inline docs/_pages/plugins/cloud/cloud-account.md]: Containership Cloud

File: docs/_pages/plugins/cloud/cloud-account-info.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
+---
+command-name: 'cloud account info'
+permalink: '/docs/plugins/cloud/cloud-account-info'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to view details about your containership cloud account.
```
Comment (normanjoyner) [inline docs/_pages/plugins/cloud/cloud-account-info.md]: Containership Cloud

File: docs/_pages/commands/svc-create.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
+---
+command-name: 'svc create'
+permalink: '/docs/commands/svc-create'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to create a new service to be run on your cluster.
+To create an service you can run the `svc create <service_name> ...flags` with any of the following flags
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-create.md]: a

File: docs/_pages/commands/svc-create.md [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+| cpus | c | Number of CPU shares to allocate the container | n | |
+|-----------------+------------+-----------------+----------------|
+| memory | m | Memory in MB allocated to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| privileged | | Whether or not the container runs in privileged mode | n | false |
+|-----------------+------------+-----------------+----------------|
+| respawn | | Whether or not the container respawns after exit | n | false |
+|=================+============+=================+================+===|
+
+> All array values, e.g. (env-var, volume, tag) can be added multiple times during an service create.
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-create.md]: a

File: docs/_pages/commands/svc-delete.md [hunk 17]
```diff
@@ -0,0 +1,18 @@
+---
+command-name: 'svc delete'
+permalink: '/docs/commands/svc-delete'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to remove an service from your cluster.
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-delete.md]: a

File: docs/_pages/commands/svc-edit.md [hunk 18]
```diff
@@ -0,0 +1,49 @@
+---
+command-name: 'svc edit'
+permalink: '/docs/commands/svc-edit'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to edit an existing service running on your cluster.
+To edit an service you can run the `svc create <service_name> ...flags` with any of the following flags
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-edit.md]: a

File: docs/_pages/commands/svc-edit.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+| cpus | c | Number of CPU shares to allocate the container | n | |
+|-----------------+------------+-----------------+----------------|
+| memory | m | Memory in MB allocated to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| privileged | | Whether or not the container runs in privileged mode | n | false |
+|-----------------+------------+-----------------+----------------|
+| respawn | | Whether or not the container respawns after exit | n | false |
+|=================+============+=================+================+===|
+
+> All array values, e.g. (env-var, volume, tag) can be added multiple times during an service edit. If
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-edit.md]: a

File: docs/_pages/commands/svc-scale-down.md [hunk 20]
```diff
@@ -0,0 +1,22 @@
+---
+command-name: 'svc scale-down'
+permalink: '/docs/commands/svc-scale-down'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to scale down an service running on your cluster.
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-scale-down.md]: a

File: docs/_pages/commands/svc-scale-up.md [hunk 21]
```diff
@@ -0,0 +1,22 @@
+---
+command-name: 'svc scale-up'
+permalink: '/docs/commands/svc-scale-up'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to scale up an service running on your cluster.
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-scale-up.md]: a

File: docs/_pages/commands/svc-create.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+|-----------------+------------+-----------------+----------------|
+| tag | t | Tag to add to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| cpus | c | Number of CPU shares to allocate the container | n | |
+|-----------------+------------+-----------------+----------------|
+| memory | m | Memory in MB allocated to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| privileged | | Whether or not the container runs in privileged mode | n | false |
+|-----------------+------------+-----------------+----------------|
+| respawn | | Whether or not the container respawns after exit | n | false |
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-create.md]: `respawn` default is `true`

File: docs/_pages/commands/svc-edit.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+|-----------------+------------+-----------------+----------------|
+| tag | t | Tag to add to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| cpus | c | Number of CPU shares to allocate the container | n | |
+|-----------------+------------+-----------------+----------------|
+| memory | m | Memory in MB allocated to the container | n | |
+|-----------------+------------+-----------------+----------------|
+| privileged | | Whether or not the container runs in privileged mode | n | false |
+|-----------------+------------+-----------------+----------------|
+| respawn | | Whether or not the container respawns after exit | n | false |
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-edit.md]: `respawn` default is `true`

File: docs/_pages/commands/svc-edit.md [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+
+> All array values, e.g. (env-var, volume, tag) can be added multiple times during an service edit. If
+you set the value of an array option to the empty string, that value will be cleared if it exists.
+
+> Tags support the dot-notation syntax for creating nested tags. e.g. (--tag metadata.hasSeen=true)
+
+Example: Editing a redis service
+
+~~~
+csctl svc create sampleredis --image=library/redis:3.0.7 -m 64 -c 0.1 -p 6379
```
Comment (normanjoyner) [inline docs/_pages/commands/svc-edit.md]: `csctl svc edit sampleredis`

File: docs/_pages/commands/host.md [hunk 25]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (normanjoyner) [inline docs/_pages/commands/host.md]: no commands are shown on this page

File: docs/_pages/commands/remote-add.md [hunk 26]
```diff
@@ -0,0 +1,19 @@
...
+`csctl remote add <remote_name> <remote_url>`
+
+|------+----+------------+----+----------|
+| Argument | Alias | Description | Required | Default |
+|-----------------|------|-----------|----------------|-------|
+| remote_name | | Name of the remote for local reference | y | |
+| remote_url | | URL of one of the `leader` nodes on your cluster | y | |
+|=================+============+=================+================+===|
+
+> This cluster will also need firewall rules for the containership api open to the machine you are using the CLI from.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-add.md]: Containership

File: docs/_pages/quick-start-guide.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+The first step is to connect your client to an existing Containership cluster with a remote by giving
+it a name and the URL to a `leader` node:
+
+> Note: `remote_name` is just a local name for you to reference this remote.
+
+`csctl remote add <remote_name> <remote_url>`
+
+> This cluster will also need firewall rules for the Containership API open to the machine you are using the CLI from.
+
+> If it is a cluster created through the Containership cloud dashboard, the `cloud` plugin can assist you in setting up remotes.
```
Comment (normanjoyner) [inline docs/_pages/quick-start-guide.md]: Cloud

File: docs/_pages/commands/remote-set-version.md.bak [hunk 28]
```diff
@@ -0,0 +1,21 @@
...
+`csctl remote set-version <remote_name> <remote_version>`
+
+|------+----+------------+----+----------|
+| Argument | Alias | Description | Required | Default |
+|-----------------|------|-----------|----------------|-------|
+| remote_name | | Name of the remote for local reference | y | |
+| remote_version | | URL of one of the `leader` nodes on your cluster | y | v1 |
+|=================+============+=================+================+===|
+
+> Currently only version `v1` exists and this will always default to the currently expected containership api version. This command
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-version.md.bak]: Containership API

File: docs/_pages/commands/remote-set-version.md [hunk 29]
```diff
@@ -0,0 +1,21 @@
...
+`csctl remote set-version <remote_name> <remote_version>`
+
+|------+----+------------+----+----------|
+| Argument | Alias | Description | Required | Default |
+|-----------------|------|-----------|----------------|-------|
+| remote_name | | Name of the remote for local reference | y | |
+| remote_version | | URL of one of the `leader` nodes on your cluster | y | v1 |
+|=================+============+=================+================+===|
+
+> Currently only version `v1` exists and this will always default to the currently expected containership API version. This command
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-version.md]: Containership

File: docs/_pages/commands/remote-set-version.md.bak [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+is only of use if you have legacy clusters you need to interface with that are on an older api version.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-version.md.bak]: API

File: docs/_pages/commands/remote-set-url.md [hunk 31]
```diff
@@ -0,0 +1,19 @@
...
+`csctl remote set-url <remote_name> <remote_url>`
+
+|------+----+------------+----+----------|
+| Argument | Alias | Description | Required | Default |
+|-----------------|------|-----------|----------------|-------|
+| remote_name | | Name of the remote for local reference | y | |
+| remote_url | | URL of one of the `leader` nodes on your cluster | y | |
+|=================+============+=================+================+===|
+
+> This cluster will also need firewall rules for the containership API open to the machine you are using the CLI from.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-url.md]: Containership

File: docs/_pages/commands/remote-set-url.md.bak [hunk 32]
```diff
@@ -0,0 +1,19 @@
...
+`csctl remote set-url <remote_name> <remote_url>`
+
+|------+----+------------+----+----------|
+| Argument | Alias | Description | Required | Default |
+|-----------------|------|-----------|----------------|-------|
+| remote_name | | Name of the remote for local reference | y | |
+| remote_url | | URL of one of the `leader` nodes on your cluster | y | |
+|=================+============+=================+================+===|
+
+> This cluster will also need firewall rules for the containership api open to the machine you are using the CLI from.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-url.md.bak]: Containership API

File: docs/_pages/commands/remote-set-version.md.bak [hunk 33]
```diff
@@ -0,0 +1,21 @@
+---
+command-name: 'remote set-version'
+permalink: '/docs/commands/remote-set-version'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to update the expected containership opensource api version
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-version.md.bak]: API

File: docs/_pages/plugins/overview.md [hunk 34]
```diff
@@ -0,0 +1,20 @@
...
+
+Plugins are extensions of the core containership functionality. There are two types of plugins:
+
+`core` - Plugins that run on the containership cluster and provide additional core functionality
+
+`cli` - Plugins that extend CLI capaiblities and commands.
+
+An example plugin is the official `containership cloud` plugin that encapsulates the additional
+functionality provided to you through the containership cloud dashboard. The plugin is both a core
+a cli plugin. The CLI plugin extends the client with a new `cloud` command that allows you to interface
```
Comment (normanjoyner) [inline docs/_pages/plugins/overview.md]: CLI

File: docs/_pages/commands/client-plugin-list.md [hunk 35]
```diff
@@ -0,0 +1,21 @@
+---
+command-name: 'client-plugin list'
+permalink: '/docs/commands/client-plugin-list'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to view a list of all the configured cli plugins installed.
```
Comment (normanjoyner) [inline docs/_pages/commands/client-plugin-list.md]: CLI

File: docs/_pages/commands/client-plugin.md [hunk 36]
```diff
@@ -0,0 +1,26 @@
+---
+command-name: 'client-plugin'
+permalink: '/docs/commands/client-plugin'
+---
+
+<h2> Overview </h2>
+
+<p>
+The {{ page.command-name }} is responsible for listing, showing, and manipulating both core plugins on your cluster and extensible cli plugins.
```
Comment (normanjoyner) [inline docs/_pages/commands/client-plugin.md]: CLI

File: docs/_pages/commands/remote-add.md [hunk 37]
```diff
@@ -0,0 +1,19 @@
+---
+command-name: 'remote add'
+permalink: '/docs/commands/remote-add'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to add a cli remote pointing to a containership cluster.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-add.md]: CLI

File: docs/_pages/commands/remote-set-url.md [hunk 38]
```diff
@@ -0,0 +1,19 @@
+---
+command-name: 'remote set-url'
+permalink: '/docs/commands/remote-set-url'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to update the url for an existing cli remote.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-url.md]: CLI

File: docs/_pages/commands/remote-set-url.md.bak [hunk 39]
```diff
@@ -0,0 +1,19 @@
+---
+command-name: 'remote set-url'
+permalink: '/docs/commands/remote-set-url'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to update the url for an existing cli remote.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-url.md.bak]: CLI

File: docs/_pages/commands/remote-set-active.md [hunk 40]
```diff
@@ -0,0 +1,31 @@
+---
+command-name: 'remote set-active'
+permalink: '/docs/commands/remote-set-active'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+The {{ page.command-name }} command allows you to update the currently active remote for your cli.
```
Comment (normanjoyner) [inline docs/_pages/commands/remote-set-active.md]: CLI

File: docs/_pages/commands/remote-add.md (hunk 26 above)
Comment (normanjoyner) [reply docs/_pages/commands/remote-add.md]: reference https://docs.containership.io/access-control/firewalls/oss-firewall-guidelines

File: docs/_pages/commands/remote-set-url.md (hunk 31 above)
Comment (normanjoyner) [reply docs/_pages/commands/remote-set-url.md]: reference https://docs.containership.io/access-control/firewalls/oss-firewall-guidelines

File: docs/_pages/quick-start-guide.md [hunk 41]
```diff
@@ -0,0 +1,142 @@
...
+~~~
+
+The first step is to connect your client to an existing Containership cluster with a remote by giving
+it a name and the URL to a `leader` node:
+
+> Note: `remote_name` is just a local name for you to reference this remote.
+
+`csctl remote add <remote_name> <remote_url>`
+
+> This cluster will also need firewall rules for the Containership API open to the machine you are using the CLI from.
```
Comment (normanjoyner) [inline docs/_pages/quick-start-guide.md]: reference https://docs.containership.io/access-control/firewalls/oss-firewall-guidelines

File: docs/_pages/commands/remote-set-url.md.bak (hunk 32 above)
Comment (normanjoyner) [reply docs/_pages/commands/remote-set-url.md.bak]: reference https://docs.containership.io/access-control/firewalls/oss-firewall-guidelines

File: docs/_pages/commands/client-plugin-add.md [hunk 42]
```diff
@@ -0,0 +1,30 @@
...
+The command supports plugins in either `npm` or `github://` format (brackets denote optional and braces denote required):
+
+`Github format: github://{organization}/{repo}[#version]`
+
+`NPM format: [@{organization}/]{repo}[@{version}]`
+
+Example adding the `cloud` plugin:
+
+~~~
+./bin/cli.js client-plugin add github://containership/containership.plugin.cloud
```
Comment (normanjoyner) [inline docs/_pages/commands/client-plugin-add.md]: replace all occurrences with `csctl`

File: docs/_pages/plugins/cloud/cloud-account-login.md [hunk 43]
```diff
@@ -0,0 +1,16 @@
+---
+command-name: 'cloud account login'
+permalink: '/docs/plugins/cloud/cloud-account-login'
+---
+
+<h2> {{ page.command-name }} </h2>
+
+<p>
+The {{ page.command-name }} allows you to logout of your cloud account for the CLI.
```
Comment (normanjoyner) [inline docs/_pages/plugins/cloud/cloud-account-login.md]: allows you to log in to

**Pull request merged**